src/Controller/CommunityController.php

```diff
@@ -18,30 +18,39 @@ class CommunityController extends AbstractController
 {
     /**
      * @Route("/", name="home")
-     * @Route("/page/{page}", name="list_page")
+     * @Route("/page={page}", name="list_page")
```

Comment:
C'est pas çc un query param. Le query param n'apparait pas dans la route et sera matérialisé par ?page=1 dans la route. Tu pourras le récupérer via l'object Request que tu peux passer en paramètre de ton action => https://stackoverflow.com/questions/9784930/how-to-get-the-request-parameters-in-symfony2

Reply:
Done